Sources/KlaviyoSwift/Vendor/ComposableArchitecture/Store.swift

```diff
@@ -398,161 +409,3 @@ final class Store<State, Action> {
 /// let store: StoreOf<Feature>
 /// ```
 typealias StoreOf<R: ReducerProtocol> = Store<R.State, R.Action>
-
-#if swift(>=5.7)
```

Comment:
I removed scoping out of this since we are using it right now and it was difficult to get to compile.